planner/nodes/report_nodes.py

```diff
@@ -0,0 +1,38 @@
+from .base_processing_node import BaseProcessingNode, ProcessingArtifact
+
+
+class ReportProcessingNode(BaseProcessingNode):
+    def __init__(self, available_artifacts, outputs):
+        super(ReportProcessingNode, self).__init__(available_artifacts, outputs)
+        self.outputs = outputs
+        self.fmt = 'json'
+
+    def get_artifacts(self):
+        datahub_type = 'derived/report'
+        resource_name = 'datapackage_report'
+        out_path = '{}.{}'.format(resource_name, self.fmt)
+        tabular_artifacts = [
+            artifact for artifact in self.available_artifacts
+                if artifact.datahub_type == 'source/tabular'
+        ]
+        output = ProcessingArtifact(
+            datahub_type, resource_name,
+            tabular_artifacts, [],
+            [('assembler.validate_resource', {}),
+             ('dump.to_path', {
```

Comment:
why do we need dump to path? 
Why not write the report directly in validate_resource?